chapters/infrastructure/overview/reading/read.md

```diff
@@ -1,62 +1,62 @@
+
+The current infrastructure components are listed below, together with roles and actions related to them:
+
+- [Git](https://git-scm.com/) is being used as the source code management tool.
+  This is used for versioned storage of content, mostly text (Markdown) and snippets (code).
+  Git is used for content development and content management.
+
+- Git is used part of the [GitHub](https://github.com/) platform, the de-facto collaborative environment for open source / content development.
+  GitHub is essential for collaborative work, allowing submissions of contributions via pull requests, issue tracking, discussions, workflow automation.
+  Content is stored as GitHub repositories.
+  It is used for content contribution and content management.
+
+- [GitHub Actions](https://github.com/features/actions) is a feature of GitHub that allows workflow automation.
+  This is used part of linting, checking and deployment workflows.
+  It is used in content development and content delivery actions.
+
+- [`reveal-md`](https://github.com/webpro/reveal-md), [`markdownpp`](https://github.com/amyreese/markdown-pp), [`ffmpeg`](https://ffmpeg.org/) are open source tools used to generate content in the corresponding format.
+  [`quiz-manager`](https://github.com/systems-cs-pub-ro/quiz-manager) is a tool developed part of Open Education Hub to convert questions in appropriate formats.
+  The initial content is generally text / ASCII or SVG.
+  The output content is HTML, PDF, XML or image / video files to be presented to the user.
+  They are used during content development, to validate content, and in content publishing and delivery.
+
+- Linters are used to validate content and ensure it obeys best practices and it is consistent throughout the repository.
+  While primarily used for source code, linters are also used for Markdown, configuration files, commits and as spellcheckers.
+  Linters are typically integrated in automated GitHub Actions workflows.
+  They are used during content development and for contributions.
+
+- [`openedu-builder`](https://github.com/open-education-hub/openedu-builder) is a component developed part of Open Education Hub to automate the publishing of a repository content as a website.
+  Generally, `openedu-builder` is invoked automatically as the publishing workflow part of GitHub Actions.
+  It can, however, be used manually, to test changes.
+  It is the prime tool used during content publishing.
+  Published content is then used and consumed directly by educators and learners.
+
+- [`vmchecker`](https://github.com/open-education-hub/vmchecker-next) is a component developed part of Open Education Hub to automate the evaluation of drills, projects and assignments.
+  It relieves educators from the repetitive task of verification, leaving them with tasks of creating content and the checkers for drills.
+  It is used as part of evaluation actions and for self-study.
```

Comment:
```suggestion
  `vmchecker` is used as part of evaluation actions and for self-study.
```
The first mention of `vmchecker` is too far away and using "it" becomes ambiguous.